fe/parameterize_lambda.py

```diff
@@ -0,0 +1,199 @@
+import argparse
+import time
+import numpy as np
+from io import StringIO
+import itertools
+import gnuplotlib as gp
+import os
+import sys
+
+from jax.config import config as jax_config
+# this always needs to be set
+jax_config.update("jax_enable_x64", True)
+
+
+from scipy.stats import special_ortho_group
+import jax
+import rdkit
+from rdkit import Chem
+from rdkit.Chem import AllChem
+
+from simtk.openmm import app
+from simtk.openmm.app import forcefield as ff
+from simtk.openmm.app import PDBFile
+
+from timemachine.lib import custom_ops, ops
+from fe.utils import to_md_units, write
+from fe import math_utils
+
+import multiprocessing
+from matplotlib import pyplot as plt
+
+from jax.experimental import optimizers
+
+from fe import simulation
+from fe import loss
+
+
+def com(conf):
+    return np.sum(conf, axis=0)/conf.shape[0]
+
+def recenter(conf, true_com, scale_factor=1):
+    mol_com = np.sum(conf, axis=0)/conf.shape[0]
+    centered = conf - mol_com  # centered to origin
+    return true_com + centered/scale_factor 
+
+from hilbertcurve.hilbertcurve import HilbertCurve
+
+def hilbert_sort(conf):
+    hc = HilbertCurve(16, 3)
+    int_confs = (conf*1000).astype(np.int64)+10000
```

Comment:
1000 and 10000 look like magic numbers and probably should be defined as constants somewhere